src/dali/dali_configuration.py

```diff
@@ -135,9 +143,25 @@ def type_check_from_string(cls, keyword: str) -> "Keyword":
     },
 }
 
-BUILTIN_CONFIGURATION_SECTIONS: Set[str] = {Keyword.TRANSACTION_HINTS.value, Keyword.IN_HEADER.value, Keyword.OUT_HEADER.value, Keyword.INTRA_HEADER.value}
+# This is a tuple, not a set, because we want this order when printed
+HISTORICAL_PRICE_KEYWORDS: Tuple[str, str, str, str, str, str] = (
```

Comment:
Yeah, you're right: all keywords are collected in one enum at the top and this is a set of values instead (like `DIRECTION_SET` or `_FIAT_FIELD_SET`. 
I suggest a couple of things:
* use dict instead of Tuple (see StackOverflow link above) or alternatively use a plain Set and just sort it on the fly when you print to the logger (either option works for me, no strong opinion)
* rename it to HISTORICAL_PRICE_KEYWORD_SET, for consistency with other similar sets in the file